### ISSUE TITLE
check_model() bugged for lmer models *only* when run as part of an RMD chunk

### ISSUE THREAD

**Issue by dpmoriarity** [easystats/performance#720]
Running check_model for a project and it works fine for lm() models in rRunning check_model for a project and it works fine for lm() models in RMD chunks but, when I transition to lmer, the figure seems bugged when I run it in an RMD chunk but works fine when it is run outside of a chunk 
 
 
![check_model broken code chunk](https://github.com/easystats/performance/assets/41660395/9afafcc4-a58c-4ef7-a6fe-4269331ba0f4)
![check_model working fine_not as part of chunk](https://github.com/easystats/performance/assets/41660395/da94d905-00cf-4cdc-8bbc-ba8b7aca4f22)

**Comment by strengejacke** [comment]
I#m not sure what the two plots mean or where they come from. What is exactly the bug you get? Do you have a reprex, e.g. the raw RMD script?

**Comment by dpmoriarity** [comment]
The top image shows the bugged output when the check_model() is ran as part of a chunk, the one below shows the non-bugged output when the exact same code is run outside of a chunk.

I've attached some data and reproducible code. oddly, this inconsistency does not come up when using data that is part of the lmer package, but it does with the project data.

I've included a) the project data and b) a knitted file with both the public data + the project data. Lmk if I can do anything else to help!

[Check_model_troubleShoot.csv](https://github.com/easystats/performance/files/15379804/Check_model_troubleShoot.csv)
[check_model_chunkedcodebug-reprex.docx](https://github.com/easystats/performance/files/15379827/check_model_chunkedcodebug-reprex.docx)
